Volume/Analytics/Analytics.swift

```diff
@@ -9,8 +9,38 @@
 import AppDevAnalytics
 
 
-/// volume-specific extensions of base Event protocol
+protocol Payload {
+    var eventName: String { get }
+    var parameters: [String: Any]? { get }
+}
+
+extension Payload {
+    var parameters: [String: Any]? {
+        nil
+    }
+}
+
+struct AnnouncementPresentedPayload: Payload {
+    let eventName = "announcement_presented"
+}
+
+class VolumeAppDevAnalytics {
 
```

Comment:
Remove new lines on 13 and 40

Reply:
Line 13 is `var eventName: String { get }`
Do you mean line 11?

Reply:
11 28 and 40. Idk where I got 13 from.